Dependency to GSSS is declared as Provided, breaks clients

Dependency to [GSSS](https://github.com/ArcBees/gsss) is declared as `provided` (see https://github.com/ArcBees/gwtchosen/blob/master/pom.xml#L237).

This requires clients of gwtchosen to declare an explicit dependency to GSSS in their poms. e.g.

```
           <dependency>
                <groupId>com.arcbees</groupId>
                <artifactId>gwtchosen</artifactId>
                <version>${gwtchosen.version}</version>
                <scope>compile</scope>
            </dependency>
            <dependency>
                <groupId>com.arcbees.gss</groupId>
                <artifactId>gsss</artifactId>
                <version>${gsss.version}</version>
                <scope>compile</scope>
            </dependency>
```

If clients of gwtchosen don't declare that dependency, Maven won't be able to locate GSSS's *.gwt.xml module files, and thus GWT compilation of client projects will fail:

```
[ERROR] Failed to execute goal org.codehaus.mojo:gwt-maven-plugin:2.7.0:compile (default) on project app: GWT Module com.arcbees.gsss.animation.Animation not found in project sources or resources.
```

--- a/plugin/pom.xml
+++ b/plugin/pom.xml
@@ -73,6 +73,10 @@
             <version>${guava.version}</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>com.arcbees.gss</groupId>
+            <artifactId>gsss</artifactId>
+        </dependency>
     </dependencies>
 
 </project>

--- a/pom.xml
+++ b/pom.xml
@@ -187,6 +187,16 @@
         </plugins>
     </build>
 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>com.arcbees.gss</groupId>
+                <artifactId>gsss</artifactId>
+                <version>${gsss.version}</version>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
     <dependencies>
         <dependency>
             <groupId>junit</groupId>
@@ -228,12 +238,6 @@
             <groupId>com.google.guava</groupId>
             <artifactId>guava-gwt</artifactId>
             <version>${guava.version}</version>
-            <scope>provided</scope>
-        </dependency>
-        <dependency>
-            <groupId>com.arcbees.gss</groupId>
-            <artifactId>gsss</artifactId>
-            <version>${gsss.version}</version>
             <scope>provided</scope>
         </dependency>
     </dependencies>

